Train_Colab.ipynb

```diff
@@ -263,50 +191,89 @@
         "\n",
         "For a more indepth Explanation of each of these paramaters check out /content/EveryDream2trainer/doc.\n",
         "\n",
-        "After youve tried a few models you will find /content/EveryDream2trainer/doc/ADVANCED_TWEAKING.md to be extremly helpful."
+        "After youve tried a few models you will find /content/EveryDream2trainer/doc/ATWEAKING.md to be extremly helpful."
```

Comment:
it should be the old link to ADVANCED_TWEAKING.md, just "ATWEAKING" is old, broken link.